Sources/Xcodeproj/generate().swift

```diff
@@ -77,10 +79,32 @@ public func generate(
     // references in the project.
     let extraDirs = try findDirectoryReferences(path: srcroot)
 
+    // Find non-source files in the source directories and root that should be added
+    // as a reference to the project.
+    var extraFiles = try findNonSourceFiles(path: srcroot)
+
+    let sourcesDirectory = srcroot.appending(component: "Sources")
```

Comment:
We should use the (root) package object to get this path as it maybe something else other than Sources.